### DISCUSSION TITLE
Firmware for RPI Pico 2

### DISCUSSION THREAD

**Discussion by opcodex64** [gusmanb/logicanalyzer#151] [Q&A] [answered]
Hello Gusman!

Thanks for your effort and thanks to everyone who support you here. 
Is the logic analyzer pre-compiled firmware already available for the RPI Pico 2 board? Maybe a beta?

**Comment by gusmanb** [comment]
There is, but it does not work with the old board, you will need the new dessign and it is still not released, I will release it soon.

Cheers.

**Reply by opcodex64** [reply]
Hello Gusman,

thanks for the very quick response. Thanks for clarification! I just was about to order some boards of the recent PCB design, but then I will wait. I don't want to be impatient, but when we can expect the release? One month ?

**Reply by gusmanb** [reply, marked as answer]
I expect to receive a new batch next week of the new boards, if you are interested contact me at logicanalyzerorders@gmail.com

Cheers.
